src/test/java/edu/ucla/library/iiif/fester/fit/PostCsvFIT.java

```diff
@@ -143,52 +165,45 @@ public final void testFullCSV(final TestContext aContext) throws CsvException, I
          * @throws CsvException If there is trouble reading the CSV data
          */
         @Test
-        @SuppressWarnings("checkstyle:indentation")
         public final void testFullCsvWithIiifHost(final TestContext aContext) {
             final Async asyncTask = aContext.async();
 
-            final String filename = ALL_IN_ONE_CSV.getName();
-            final String pathname = ALL_IN_ONE_CSV.getAbsolutePath();
-            final MultipartForm form = MultipartForm.create().textFileUpload(Constants.CSV_FILE, filename, pathname,
-                    Constants.CSV_MEDIA_TYPE).attribute(Constants.IIIF_HOST, ImageInfoLookup.FAKE_IIIF_SERVER);
```

Comment:
Not all of the tests had been using a form with the `IIIF_HOST` attribute set, but now they are. Is that intentional?

Reply:
Not all of them needed that because not all tested images. It doesn't hurt if they have it, though, even if they don't absolutely need it. It would always be supplied by the form if someone was manually running the tests.

Reply:
Want to remove `testFullCsvWithIiifHost` then? It's now identical to `testFullCSV`.

Reply:
Also it looks like the `throws` declarations on `testFullCSV` are no longer needed.

Reply:
Okay, done. Good catch.